tests/restarting/to_7.0.0/CycleTestRestart-1.txt

```diff
@@ -30,4 +32,4 @@ testTitle=Clogged
 
     testName=SaveAndKill
     restartInfoLocation=simfdb/restartInfo.ini
-    testDuration=10.0
\ No newline at end of file
+    testDuration=10.0
```

Comment:
Is the current behavior that test never timeout?

Reply:
This appears to be a whitespace change.